lib/tasks/growstuff.rake

```diff
@@ -4,18 +4,20 @@ namespace :growstuff do
   # usage: rake growstuff:admin_user name=skud
 
   task admin_user: :environment do
-    member = Member.find_by_login_name(ENV['name']) or raise "Usage: rake growstuff:admin_user name=whoever (login name is case-sensitive)"
-    admin  = Role.find('admin')
-    member.roles << admin
+    add_role_to_member! ENV['name'], 'admin'
   end
 
   desc "Add a crop wrangler user, by name"
   # usage: rake growstuff:cropwrangler_user name=skud
 
   task cropwrangler_user: :environment do
-    member = Member.find_by_login_name(ENV['name']) or raise "Usage: rake growstuff:cropwrangler_user name=whoever (login name is case-sensitive)"
```

Comment:
Any change you can retain the raise 'how to use this' messaging or shift it into the desc?